### PR TITLE
fix: asset object key TDE-188

### PR DESCRIPTION
**Pull request by amfage**
[https://linzgovt.atlassian.net/browse/TDE-188](https://linzgovt.atlassian.net/browse/TDE-188)
The Asset Object key was set to the media type.
This is an interim change (until we know more about the assets there will be) to set it to "visual" if the type is geotiff, then to follow the previous logic and set to the file extension if there is no media type. Then to the checksum if there is no extension.

### CODE REVIEW

File: topo_processor/stac/asset.py [hunk 1]
```diff
@@ -41,6 +41,16 @@ def get_checksum(self) -> str:
             self.properties["file:checksum"] = multihash_as_hex(self.source_path)
         return self.properties["file:checksum"]
 
+    def get_key(self) -> str:
+        if self.content_type:
```
Comment (paulfouquet) [inline topo_processor/stac/asset.py:45]: You might not need to test if `content_type` is not `None`

**Review by paulfouquet — APPROVED** [review]
Looks good to me, just a suggestion

**Comment by amfage** [comment]
Change of approach needed.
